src/highlevel.jl

```diff
@@ -102,12 +102,37 @@ function create_executable(device, entry, obj; globals=())
     return ROCExecutable(device, data, entry; globals=globals)
 end
 
+function create_kernel_queue(;
```

Comment:
```suggestion
function get_kernel_queue(;
```
We shouldn't actually be creating a new queue here, only getting the one specified by the user.